gradle.properties

```diff
@@ -1,4 +1,4 @@
 # Sets default memory used for gradle commands. Can be overridden by user or command line properties.
 # This is required to provide enough memory for the Minecraft decompilation process.
 org.gradle.jvmargs=-Xmx3G
-org.gradle.daemon=false
\ No newline at end of file
+org.gradle.daemon=false
```

Comment:
the hell happened here???

Reply:
I accidentally committed a local change and quickly undid it, forgetting about the blank line below it. There's no functional change to this file

Reply:
I still think it might be worth putting in the blank line as it was so that it doesn't unnecessarily change this file, but it's not a big deal so I'll just give it the thumbs up anyway.